src/morphodict/site/settings.py

```diff
@@ -283,26 +242,28 @@ def defaultDatabasePath():
 
 USE_TZ = True
 
-############################## API app settings ###############################
-
-# We only apply affix search for user queries longer than the threshold length
-AFFIX_SEARCH_THRESHOLD = 4
 
-############################## staticfiles app ###############################
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/3.2/howto/static-files/
 
 STATIC_URL = env("STATIC_URL", "/static/")
 
-STATIC_ROOT = os.fspath(env("STATIC_ROOT", default=BASE_PATH / "static"))
+STATIC_ROOT = os.fspath(env("STATIC_ROOT", default=BASE_DIR / "collected-static"))
 
 if DEBUG:
     # Use the default static storage backed for debug purposes.
     STATICFILES_STORAGE = "django.contrib.staticfiles.storage.StaticFilesStorage"
 else:
     # In production, use a manifest to encourage aggressive caching
-    # Note requires `python manage.py collectstatic`!
+    # Note requires `manage.py collectstatic`!
     STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"
 
-######################################## logging ###############################
+# Default primary key field type
+# https://docs.djangoproject.com/en/3.2/ref/settings/#default-auto-field
+
+DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
```

Comment:
I don't understand why this was added. Could you please explain?

Reply:
This is a new setting automatically added by `startproject` in Django 3.2. I believe it is part of an effort to make primary keys at least 64 bits for new projects while preserving backwards-compatibility. Otherwise bad things happen PostgreSQL when you have 32-bit primary keys and you [run out or roll over](https://tech.coffeemeetsbagel.com/reaching-the-max-limit-for-ids-in-postgres-6d6fa2b1c6ea).